README.md

```diff
@@ -51,6 +51,67 @@ yarn start:backend # Starts on http://localhost:3001
 yarn start:frontend # Starts on http://localhost:3000
 ```
 
+### How to use Snaplet seeding
+
+You can use the seeding script to populate your database with realistic mock data using the Snaplet Seed library and its `copycat` function.
+
```

Comment:
You don't have to describe here how to set up a project with snaplet. Instead, you should describe how a new developer can use snatplet to seed their database in this project. If I recall correctly, they basically have to run `npx @snaplet/seed sync´ and `yarn prisma db seed` from the backend dir